scripts/make.py

```diff
@@ -160,7 +160,7 @@ def build_native_tools(self) -> None:
         dist_comps += self.llvm_binutils
 
         cmake_defs = {
-            'LLVM_TARGETS_TO_BUILD:STRING': 'ARM',
+            'LLVM_TARGETS_TO_BUILD:STRING': 'ARM;AArch64',
```

Comment:
Please enable AArch64 conditionally (depending on `cfg.variants`).

Reply:
Will do. I assume that both the ARM and AArch64 backends should be made conditional, based on whether they are needed.

Reply:
IIRC, an AArch64-only configuration will fail at build time because the AArch64 backend uses some functions from the Arm backend. However, I am not 100% certain.

Reply:
I implemented both ARM and AArch64 backends to be conditional. I didn't see any failures when the ARM backend wasn't present for an aarch64-only toolchain.